src/components/existing-request/existing-request-display.vue

```diff
@@ -726,6 +721,8 @@ export default class ExistingRequestDisplay extends Mixins(
         // fetch updated payments
         await this.fetchNrPayments(nrId)
       }
+
+      // get the first pending payment
       this.pendingPayment = this.payments.find(
         payment => (
           ![PaymentStatus.APPROVED, PaymentStatus.COMPLETED, PaymentStatus.CANCELLED, PaymentStatus.REFUND_REQUESTED]
```

Comment:
I know you commented above but i don't full understand. 
Can there be multiple pending payments? 
If so, only the first one matters?

Reply:
No, there should not be multiple pending payments.

However, if (eg, during local testing) we do have more than one, we might expect the last payment to be retried, but it's actually the first payment that gets retried. It confused me when I bumped into this so I added the comment.

Reply:
Ok sounds good, thanks!